### DISCUSSION TITLE
Add my game "https://guess-the-year.davjhan.com/"

### DISCUSSION THREAD

**Discussion by davjhan** [janosh/awesome-sveltekit#4] [Ideas]
Hey! I was wondering if it would be suitable to add my game to this repo.

It uses Sveltekit, and tailwindcss.

It builds to a static site and is hosted on Netlify.

The source can be found here:
https://github.com/davjhan/guess-the-year-game

**Comment by janosh** [comment]
Certainly, please submit a PR! I really enjoyed playing your game!

**Comment by erwingame** [comment]
Interesting!
